amivapi/events/model.py

```diff
@@ -257,7 +257,6 @@
                 'description': 'Start time of the event itself. If you define '
                                'a start time, an end time is required, too.',
                 'example': '2018-10-17T18:00:00Z',
-
```

Comment:
Is it necessary to remove this newline? I think it helped the readability.

Reply:
Ah... for me it is the contrary, it is disturbing, as it should be part of the same json structure on the same level, I thought that it was clearer to keep it on one level.

Reply:
In this case I would add the line again, as this has nothing to do with the feature of this PR.